<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserLogin</class>
 <widget class="QDialog" name="UserLogin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>239</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>239</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>239</height>
   </size>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>321</width>
     <height>52</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="confirmNum_layout" stretch="0,0,0">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <item>
     <widget class="QLabel" name="label_authCode">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>ResStr_Verification_Code</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="le_autoCode">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>28</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
	border-top-left-radius:12px;
	border-bottom-left-radius:12px;
}</string>
      </property>
      <property name="maxLength">
       <number>6</number>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="bt_getAuthCode">
      <property name="minimumSize">
       <size>
        <width>115</width>
        <height>28</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>115</width>
        <height>28</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color:rgb(179, 179, 179);
	border-top-right-radius:12;
	border-bottom-right-radius:12;
	color: brack;
}
QPushButton:hover{
	background-color:rgb(176, 176, 176);
	border-top-right-radius:12;
	border-bottom-right-radius:12;
	color: green;
}
QPushButton:pressed{
	background-color:rgb(76, 76, 76);
	border-top-right-radius:12;
	border-bottom-right-radius:12;
	color: white;
}</string>
      </property>
      <property name="text">
       <string>ResStr_Get_Verification_Code</string>
      </property>
      <property name="autoDefault">
       <bool>true</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="bt_login">
   <property name="geometry">
    <rect>
     <x>126</x>
     <y>190</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	border-image: url(:/Images/Btn_Green.png);
	color: rgb(255, 255, 255);
}
QPushButton:hover{
	border-image: url(:/Images/Btn_Green.png);
	color: brack;
}
QPushButton:pressed{
	border-image: url(:/Images/Btn_Green_Pressed.png);
	color: white;
}</string>
   </property>
   <property name="text">
    <string>ResStr_Login_</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_title">
   <property name="geometry">
    <rect>
     <x>66</x>
     <y>30</y>
     <width>261</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ResStr_Express_Login_using_mobile_number</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labMsg">
   <property name="geometry">
    <rect>
     <x>26</x>
     <y>160</y>
     <width>341</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
color:red;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>321</width>
     <height>42</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="userName_layout">
    <property name="spacing">
     <number>0</number>
    </property>
    <item>
     <widget class="QLabel" name="label_userName">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>ResStr_Account</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="le_userName">
      <property name="minimumSize">
       <size>
        <width>180</width>
        <height>28</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
	border-radius:12px;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="maxLength">
       <number>11</number>
      </property>
      <property name="placeholderText">
       <string>ResStr_Msg_4</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>le_userName</tabstop>
  <tabstop>le_autoCode</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
